Update manipulating entities tutorial with cascade delete

deakgeek opened the pull request:
#98

deakgeek committed:
Updated the manipulating entities tutorial to include cascade deletion

deakgeek committed:
Updated wording

helmoski commented on content/efcore/getting-started/manipulating-entities.md:
```diff
@@ -132,7 +132,4 @@ using (var context = new LibraryContext())
- 
-> **Note** {.note}  
-> The Configuring Deletion lesson has not been added yet. For now, please refer to the [Cascade Delete](https://docs.microsoft.com/en-us/ef/core/saving/cascade-delete) page in the docs.
+In the previous example, we were able to successfully delete a book, but what happened to that book's checkout records? The short answer is it depends. The way that EF Core handles deletion of an entity with related data depends on several factors including whether the related data is tracked, how EF Core is configured, and how the database is configured. By default, EF Core will delete any track ...
```
Add quotes around cascade delete docs page name

arynhuck replied:
🌮

arynhuck committed:
Quotes around docs page name

Keeping things standard, ya know? This is how we've had all the other
articles in the docs referred to.

arynhuck committed:
merge master -> branch

Pulled in master to prepare for merging of this branch into master.

arynhuck commented:
:shipit: ?

arynhuck commented:
@helmoski :shipit: :shipit: :shipit:
